src/dpp/guild.cpp

```diff
@@ -122,6 +122,10 @@ guild_member& guild_member::set_communication_disabled_until(const time_t disabl
 	this->communication_disabled_until = disabled_timestamp;
 	return *this;
 }
+	
+bool guild_member::operator == (guild_member const& other_member) const {
+	return this->user_id == other_member.user_id && this->guild_id == other_member.guild_id;
```

Comment:
It should return false if the ids are 0. Because if someone is creating two guild_member objects, it returns true when compared because they have the default value 0.

Reply:
Alright, will edit accordingly.